### DISCUSSION TITLE
Nevermind (Please Delete)

### DISCUSSION THREAD

**Discussion by chuysbz** [spatie/laravel-medialibrary#2112] [Q&A] [answered]
Hello,

Is there a way to return the uploaded URL?

I'm adding the media from request like this: `$model->addMediaFromRequest('media');`. Now what I need to do is to return the added media full url but I can't find the right way to do it.

Is this possible?

Thanks in advance.

**Comment by freekmurze** [comment, marked as answer]
Try this:

```php
$model
   ->addMediaFromRequest('media')
   ->toMediaCollection()
   ->getUrl();
```
